client/src/App.tsx

```diff
@@ -121,6 +128,10 @@ const Routes = () => (
 
 class App extends React.Component {
   public render() {
+    window.Intercom("boot", {
+      app_id: "gmkvd6s1",
```

Comment:
Maybe put this in `config.js`? Not a big deal either way.

Reply:
Looks like we don't have a config.js/config.json for the client. Seems like a candidate for a future refactor.